app/controllers/ChoiceController.scala

```diff
@@ -88,8 +88,6 @@ class ChoiceController @Inject()(
                   Redirect(controllers.routes.CancelDeclarationController.displayForm())
                 case Submissions =>
                   Redirect(controllers.routes.SubmissionsController.displayListOfSubmissions())
-                case _ =>
-                  Redirect(controllers.routes.ChoiceController.displayPage())
```

Comment:
There is no possibility to achieve this case. Form validation will block this. Should we stay with this or removing is okay?

Reply:
Will we get a warning if it is removed? if so can we suppress it?

Reply:
No, there is no warning